test/transmuxer.test.js

```diff
@@ -3412,7 +3412,7 @@ QUnit.test('generates a video init segment', function() {
   QUnit.ok(segments[0].info, 'video info is alongside video segments/bytes');
 
   mp4VideoProperties.forEach(function(prop) {
-    QUnit.ok(segments[0].info[prop], 'video info has ' + prop);
+    QUnit.ok(prop in segments[0].info, 'video info has ' + prop);
```

Comment:
This was the easiest way to "fix" this test when sarRatio is undefined. Is this acceptable?

Reply:
I don't know for sure, but I'd imagine that this checks to make sure the MP4 init segment has all the properties passed through from the PES. So this change would mean the test is no longer testing for what it should.

Could you add the SAR data to the SPS data?

Reply:
> I don't know for sure, but I'd imagine that this checks to make sure the MP4 init segment has all the properties passed through from the PES. So this change would mean the test is no longer testing for what it should.

mp4VideoProperties are from the SPS, not the PES header (if that's what you mean). I changed it back to how it was for every other property. I'm not sure how else to do it. (https://github.com/videojs/mux.js/pull/275/commits/2c2776277b356db5c177cadfeecf2581450c7854)

> Could you add the SAR data to the SPS data?

I'm not sure what you mean by this?